### PR TITLE
[PAY-1389] CTA on create chat when not permitted

### PR DESCRIPTION
**Pull request by dharit-tan**
### Description
Add CTA in create chat flow that tells user what action to take in order to message the unpermitted user. Opens InboxUnavailable drawer.

### How Has This Been Tested?

Local ios stage

Simulator Screen Shot - iPhone 14 Pro - 2023-06-11 at 21 02 45
Simulator Screen Shot - iPhone 14 Pro - 2023-06-11 at 21 02 41

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

**Review by sddioulde — APPROVED** [review]
Lgtm. Although slightly concerned about the follows you overlap

File: packages/mobile/src/screens/chat-screen/ChatUserListItem.tsx [hunk 1]
```diff
@@ -95,45 +110,71 @@ const useStyles = makeStyles(({ spacing, palette, typography }) => ({
...
     >
       <View style={styles.border}>
-        <View
-          style={[styles.userContainer, !canCreateChat ? styles.dim : null]}
-        >
-          <ProfilePicture profile={user} style={styles.profilePicture} />
+        <View style={styles.userContainer}>
+          <ProfilePicture
+            profile={user}
+            style={[styles.profilePicture, !canCreateChat ? styles.dim : null]}
```
Comment (rickyrombo) [inline packages/mobile/src/screens/chat-screen/ChatUserListItem.tsx:176]: Just double checking - do we still dim this?

**Review by rickyrombo — APPROVED** [review]

File: packages/mobile/src/screens/chat-screen/ChatUserListItem.tsx (hunk 1 above)
Comment (dharit-tan) [reply packages/mobile/src/screens/chat-screen/ChatUserListItem.tsx:204]: We dim just the profile picture (according to figma)

**Comment by dharit-tan** [comment]
> Lgtm. Although slightly concerned about the follows you overlap

Turns out we hide the follows you tag when we show the cta.

**Pull request merged**